### ISSUE TITLE
How do we properly display the the language in the language it was meant to be in?

### ISSUE THREAD

**Issue by rlam3** [marmelroy/Localize-Swift#72]
How do we properly display the the language in the language it was meant to be in?

Instead of when set to english／Base... 
Chinese
English 

It would be ...
中文（繁體）
English

**Comment by rlam3** [comment]
Also I was wondering if it was possible to update the entire app to be localized through using Localize-Swift? Thanks.

**Comment by galahador** [comment]
@rlam3  Yes it's possible, add to all strings .localize() and set button to change language when tapped.

**Comment by rlam3** [comment]
@galahador does this affect the main OS language setting?

**Comment by galahador** [comment]
@rlam3 nope. It was my bad. I set.localize() on NSLocalizeString.... when you change a language, to German, it won't work.
